src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -1205,511 +1143,459 @@ namespace CondenserLoopTowers {
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(17) = -0.00270968955115031;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(18) = 0.00112277498589279;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(19) = -0.00127758497497718;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(20) = 0.0000760420796601607;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(21) = 1.43600088336017;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(22) = -0.5198695909109;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(23) = 0.117339576910507;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(24) = 1.50492810819924;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(25) = -0.135898905926974;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(26) = -0.152577581866506;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(27) = -0.0533843828114562;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(28) = 0.00493294869565511;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(29) = -0.00796260394174197;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(30) = 0.000222619828621544;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(31) = -0.0543952001568055;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(32) = 0.00474266879161693;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(33) = -0.0185854671815598;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(34) = 0.00115667701293848;
-                state.dataCondenserLoopTowers->towers(state.dataCondenserLoopTowers->towers(TowerNum).VSTower).Coeff(35) = 0.000807370664460284;
+                state.dataCondenserLoopTowers->towers(tower.VSTower).Coeff(1) = 0.52049709836241;
+                state.dataCondenserLoopTowers->towers(tower.VSTower).Coeff(2) = -10.617046395344;
```

Comment:
Also, this coefficients should be in a `constexpr std::array`. We should not be assigning them at runtime.